lib/phone_shop/widgets/components/shop_card.dart

```diff
@@ -0,0 +1,250 @@
+import 'package:flutter/material.dart';
+
+class ShopCard extends StatelessWidget {
+  final String title;
+  final int price;
```

Comment:
Why is price type int and still you display it as double with .00. Make it double.